netbench/netbench-driver/etc/Dockerfile

```
@@ -67,25 +67,25 @@ RUN set -eux; \
 
 RUN set -eux; \
   cd netbench; \
-  cargo run --bin netbench-scenarios; \
+  cargo build --bin netbench-scenarios; \
+  ./target/debug/netbench-scenarios --request_response.count 10 --request_response.request_size 100KB --request_response.response_size 10MB; \
```

I think for the initial version it would be better to do a single, large request:

```suggestion
  ./target/debug/netbench-scenarios --request_response.response_size 1GB; \
```